src/app/core/notifications/notifications_manager.nim

```diff
@@ -13,6 +13,7 @@ logScope:
 const NOTIFICATION_SOUND = "qrc:/imports/assets/audio/notification.wav"
 
 # Signals which may be emitted by this class:
+const SIGNAL_ADD_NOTIFICATION_TO_ACTIVITY_CENTER* = "addNotificationToActivityCenter"
 const SIGNAL_DISPLAY_APP_NOTIFICATION* = "displayAppNotification"
```

Comment:
Similar as above, according to [new message notification diagram](https://drive.google.com/file/d/1L_9c2CMObcDcSuhVUu97s9-_26gtutES/view) and [other actions diagram](https://drive.google.com/file/d/1KmG7lJDJIx6R_HJWeFvMYT2wk32RoTJQ/view) we should display ephemeral app notification in some cases, that's the reason why we have `SIGNAL_DISPLAY_APP_NOTIFICATION ` signal, which is emitted whenever we need to display ephemeral notification, so all we need to do is just to connect to this signal and display it.

This part is not done here cause we don't have yet ephemeral notifications developed.
We will handle this through the new issue.